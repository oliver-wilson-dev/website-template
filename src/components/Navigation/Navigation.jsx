import React from 'react';
import TopNav from './TopNav';
import styles from './Navigation.module.css';
import SideNav from './SideNav';

const Navigation = () => (
  <div className={styles.navigation}>
    <TopNav />
    <SideNav />
  </div>
);

export default Navigation;
