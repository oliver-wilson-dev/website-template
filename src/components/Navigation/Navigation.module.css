@import "../../styles/globalVars.css";

.navigation {
  height: $navigation-height;
  position: fixed;
  top: 0;
  background-color: #f8f8f8;
  width: 100%;
  padding: 0px;
  margin: 0px;
  box-shadow: 0px 0px 8px #000000;
  display: flex;
  justify-content: center;
  align-items: center;
}