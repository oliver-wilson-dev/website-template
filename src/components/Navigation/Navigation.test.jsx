import React from 'react';
import { shallow } from 'enzyme';
import Navigation from './Navigation';

jest.mock('./SideNav', () => {
  const SideNav = () => null;

  return SideNav;
});

jest.mock('./TopNav', () => {
  const TopNav = () => null;

  return TopNav;
});

const render = () => shallow(<Navigation />);

describe('Navigation', () => {
  it('renders correctly', () => {
    expect(render()).toMatchSnapshot();
  });
});
