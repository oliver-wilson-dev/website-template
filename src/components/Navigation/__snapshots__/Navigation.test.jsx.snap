// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders correctly 1`] = `
<div
  className="navigation"
>
  <TopNav />
  <SideNav />
</div>
`;
